src/callbacks/liquidity/BaselineV2/BaselineAxisLaunch.sol

```diff
@@ -446,6 +465,17 @@ contract BaselineAxisLaunch is BaseCallback, Policy, Owned {
             // Anchor range lower is the anchor tick width below the anchor range upper
             int24 anchorRangeLower = anchorRangeUpper - cbData.anchorTickWidth * tickSpacing;
 
+            // Validate that the pool target tick is within the anchor range
+            // The `_onSettle()` function will perform swaps (if necessary) to move the pool tick to the target tick in order to mitigate manipulation.
+            // The value must be in the anchor range and below the anchor upper tick (which is where the discovery range begins).
+            if (
+                cbData.poolTargetTick < anchorRangeLower
+                    || cbData.poolTargetTick >= anchorRangeUpper
+            ) {
+                revert Callback_Params_InvalidPoolTargetTick(anchorRangeLower, anchorRangeUpper);
+            }
+            poolTargetTick = cbData.poolTargetTick;
+
             // Set the anchor range
```

Comment:
another useful check would be to ensure the pools active tick == poolTargetTick

Reply:
but should be fine without it.

Reply:
This can be DoS'd by moving it prior to auction creation, which is the same reason we pass it in instead of reading from the pool.